ironfish-cli/networks.json

```diff
@@ -0,0 +1,105 @@
+{
+  "1": {
+    "button_colour": "yellow",
+    "minimum_confirmation": 8,
+    "explorer_url": "https://testnet.bscscan.com/",
+    "public_rpc": "https://nd-341-988-450.p2pify.com/b029215107a07425b50f0c0f10efba76",
+    "chainport_network_id": 1,
+    "shortname": "BNB",
+    "name": "BSC",
+    "utils_contract": "0xb8e4612f68401af5d6f1a230df653c559207fd60",
+    "chain_id": 97,
```

Comment:
Just a reference in case you are interested to see the information we get back from chainport